### PR TITLE
test: add test for vlan.id - v3

### PR DESCRIPTION
**Pull request by AkakiAlice**
Ticket: #1065

Description:
- Add Suricata-Verify test for vlan.id keyword

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/1065

Suricata PR: https://github.com/OISF/suricata/pull/12103

Previous PR: https://github.com/OISF/suricata-verify/pull/2124

### CODE REVIEW

File: tests/detect-vlan-id/test.rules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+alert ip any any -> any any (msg:"Vlan ID is equal to 200 with especific layer"; vlan.id:200,1; sid:1;)
```
Comment (catenacyber) [inline tests/detect-vlan-id/test.rules:1]: nit typo `especific`

File: tests/detect-vlan-id/test.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+args:
+- -k none
+
+checks:
+- filter:
+   count: 1
+   match:
+     event_type: alert
+     alert.signature_id: 1
```
Comment (catenacyber) [inline tests/detect-vlan-id/test.yaml:12]: Could we check the vlan id in the alert data ?

Comment (AkakiAlice) [reply tests/detect-vlan-id/test.yaml:12]: Sorry, I didn't understand the question. Could you be more specific, please?

Comment (catenacyber) [reply tests/detect-vlan-id/test.yaml:12]: Can we we add in this match section a check about `vlan.id: 200` ? (with the good syntax and value)

Comment (AkakiAlice) [reply tests/detect-vlan-id/test.yaml:12]: I don't think so. Should I use these other commits as examples for the implementation?
- https://github.com/OISF/suricata/pull/11945/commits/a8fe7a54e70c4d19c4c2a0accd0e38996f023341
- https://github.com/OISF/suricata-verify/pull/2088/commits/e5861a895ceb07b6af667052156a923309983132

Comment (catenacyber) [reply tests/detect-vlan-id/test.yaml:12]: schema.json tells me that it should be `vlan[0]: 200`

**Pull request closed without merging**